app/scripts/controllers/preferences.js

```diff
@@ -41,6 +41,9 @@ export default class PreferencesController {
       useNftDetection: false,
       useCurrencyRateCheck: true,
       openSeaEnabled: false,
+      ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+      securityAlertsEnabled: false,
```

Comment:
Do we want this to be so abstract in case we support other mechanisms or should we be explicit and call this `blockaidEnabled`?

Reply:
Yep there might be more security providers in future. Also product team suggested name `security_alerts_enabled` for metrics, so I preferred to use that name at all places.